doc/usage/configuration.rst

```diff
@@ -2130,13 +2161,69 @@ information.
         .. versionadded:: 1.2
 
      ``'fontenc'``
-        "fontenc" package inclusion, default ``'\\usepackage[T1]{fontenc}'``.
+        "fontenc" package inclusion, defaults to
+        ``'\\usepackage[T1]{fontenc}'``.
 
         .. versionchanged:: 1.5
            Defaults to ``'\\usepackage{fontspec}'`` when
            :confval:`latex_engine` is ``'xelatex'``.
         .. versionchanged:: 1.6
            ``'lualatex'`` also uses ``fontspec`` per default.
+        .. versionchanged:: 2.0
+           With ``'pdflatex'`` you can add ``LGR`` and/or ``T2A``
+           (before ``T1`` which should remain the last) to trigger
+           automatic support of occasional Greek and Cyrillic letters
+           in text.
+
+           .. attention::
+
+              Prior to 2.0, Unicode Greek letters were escaped to use LaTeX
+              math mark-up. This is not the case anymore so it may be needed
+              to modify this key into ``'\\usepackage[LGR,T1]{fontenc}'`` and
+              also to make sure to have the suitable Greek font packages
+              as listed in :doc:`../changes` (or replacements).
+
+     ``'textgreek'``
+        The default (``'pdflatex'`` only) is
+        ``'\\usepackage{textalpha}'``, but only if ``'fontenc'`` was
+        modified by user to include ``LGR`` option.  If not, the key
+        value will be forced to be empty string.
+
+        This is needed for ``pdfLaTeX`` to support Unicode input of Greek
+        letters such as φύσις. Expert users may want to load the ``textalpha``
+        package with its option ``normalize-symbols``.
```

Comment:
the only reason for documenting this key is to allow usage with this extra option